.ci/scripts/check_click_for_mypy.py

```diff
@@ -1,9 +1,12 @@
 #!/bin/env python3
 
-import click
+from importlib.metadata import version
+
 from packaging.version import parse
 
-if parse(click.__version__) < parse("8.1.1") or parse(click.__version__) >= parse("8.2"):
```

Comment:
click 8.2 was just released, our pyproject.toml only requests for <9, so this fails now

Reply:
tbf they didn't actually remove it, but the warning alone caused our CI to fail